src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionsHandlerBase.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
At this point is the entire project #nullable enabled? Want to just set it in the project file?

Reply:
There's a few tricky bits yet - I'll do that in a followup.